debian/rules

```diff
@@ -2,5 +2,9 @@
 
 BUILDDIR=build_dir
 
+# Ignore false warning with GCC < 11.3: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=99578
+# Reference OpenRCT2 bug report: https://github.com/OpenRCT2/OpenRCT2/issues/16691
+export DEB_CFLAGS_MAINT_APPEND = -Wno-stringop-overflow
```

Comment:
```suggestion
export DEB_CXXFLAGS_MAINT_APPEND = -Wno-stringop-overflow
```

Have you verified those changes on your end? I only ever build packages through launchpad, haven't set up my system for proper deb building.

Reply:
The documentation on those variables is pretty obscure, please verify which of the options should be used

Reply:
Given the lack of good documentation, I just made it set both variables.

I haven't been able confirm the fix locally, because my system has a newer version of GCC that isn't affected by the bug.

Reply:
Fair